test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -76,6 +76,34 @@
                    {:resid "z"}]
         counts (frequencies (map :resid resources))]
     (testing "resource-label"
+      (is (= ["y ([:t.administration/org]: NBN en) ([:t.administration/licenses]: Performance License, Second License)"
+              "y ([:t.administration/licenses]: Performance License)"]
+             (map #(resource-label % :en counts) [{:resid "y"
```

Comment:
Why still the `map` here? Shouldn't the second be its own case like the second `is` below?

Reply:
I wanted to check the case with two similar resids. 
I can add another case below.

Reply:
One `is` per one call of tested function i.e. `resource-label` is the normal case. Also now you already moved half a step into this direction but not quite there.